Layout w HTML i CSS/product_page/solution.md

```diff
@@ -0,0 +1,12 @@
+1. Stworzenie dwóch głównych elementów strony: <header> i <main>
+
+2. W <header> znajdą dwa <div>, w pierwszym będzie logo (sneakers) i menu (collections.... Contact), a w drugim ikonka wózka i użytkownika.
+
+3. W cześci <main> zbudujemy dwa <div>. Pierwszą to będzie galeria  
+   a druga będzie miała opis produktu z dwoma elementami.
+
+4. Pierwszy <div> będzie zawiarał 3 elementy i umożliwi wybranie  
+    ilości produktu
+
+5. Drugi elemento to <button> i pozwoli na dodanie produktu do
+   koszyka
```

Comment:
Każdy z tych punktów można potraktować jako bazę do rozpisania zadań/-nia, więc na pewno jest to użyteczne. Jeśli znajdziecie chwilę, zastanówcie się, jak podzielić sobie stronę na zadania używając powyższych notatek, ale pomijając szczegóły implementacyjne (implementacja to raczej wybór dewelopera aniżeli narzut z zadania :) )